cluster-operator/src/main/java/io/strimzi/operator/cluster/model/Storage.java

```diff
@@ -177,7 +177,7 @@ public static StorageType from(String type) {
             } else if (type.equals(LOCAL.type)) {
                 return LOCAL;
             } else {
-                throw new IllegalArgumentException("Unknown type: " + type);
+                throw new IllegalArgumentException("Unknown type: " + type + ". Allowed types are: " + EPHEMERAL + ", " + PERSISTENT_CLAIM + " and " + LOCAL + ".");
```

Comment:
Maybe we can use the implicit `values()` method in the `StorageType` enum for printing the available types ?

Reply:
Sure. Thanks!